assets/src/edit-story/app/canvas/useCanvasKeys.js

```diff
@@ -252,6 +260,28 @@ function useCanvasKeys(ref) {
   }, [addPastedElements, selectedElements, selectedElementAnimations]);
 
   useGlobalKeyDownEffect('clone', () => cloneHandler(), [cloneHandler]);
+
+  const isPlaying = [
+    STORY_ANIMATION_STATE.PLAYING,
+    STORY_ANIMATION_STATE.PLAYING_SELECTED,
+  ].includes(animationState);
+  useGlobalKeyDownEffect(
+    { key: ['mod+k'] },
+    (evt) => {
+      evt.preventDefault();
+
+      updateAnimationState({
+        animationState: isPlaying
+          ? STORY_ANIMATION_STATE.RESET
+          : STORY_ANIMATION_STATE.PLAYING,
+      });
+
+      trackEvent('canvas_play_animations', {
```

Comment:
nit: can you add this to the tracking question section of the PR description template?  ("Does this PR change what data or activity we track or use?")